src/common/containers/GameArea.js

```diff
@@ -251,9 +187,6 @@ export class GameArea extends Component {
   componentDidMount() {
```

Comment:
Why do we have a `componentDidMount` step in both the component and the container? I guess I'm not understanding why likes 254:256 were moved.

Reply:
I split the original `componentDidMount` into a rendering part and a routing part, and assigned them to the component and container, respectively.